sentry-core/src/hub.rs

```diff
@@ -284,23 +286,20 @@ impl Hub {
         }}
     }
 
+    /// Capture any `std::error::Error`.
+    pub fn capture_error<E: Error + ?Sized>(&self, error: &E) -> Uuid {
+        let event = event_from_error(error);
+        self.capture_event(event)
+    }
+
     /// Captures an arbitrary message.
     pub fn capture_message(&self, msg: &str, level: Level) -> Uuid {
-        with_client_impl! {{
-            self.inner.with(|stack| {
-                let top = stack.top();
-                if let Some(ref client) = top.client {
-                    let mut event = Event {
-                        message: Some(msg.to_string()),
-                        level,
-                        ..Default::default()
-                    };
-                    self.capture_event(event)
-                } else {
-                    Uuid::nil()
-                }
-            })
-        }}
+        let mut event = Event {
+            message: Some(msg.to_string()),
+            level,
+            ..Default::default()
+        };
```

Comment:
This runs now even if the client is disabled. Is this intended? Ideally, we would skip this if there is no client bound.